TwilioVerifySDK/TwilioVerify/Sources/Data/Storage.swift

```diff
@@ -107,6 +110,8 @@ private extension Storage {
         break
       }
     }
+    updateVersion(version: Constants.version)
```

Comment:
This should be `version` like the one in the guard of above.
https://github.com/twilio/twilio-verify-ios/blob/d45d9537df35d8765b2ea810da146ce5b65b1e08/TwilioVerifySDK/TwilioVerify/Sources/Data/Storage.swift#L92

Reply:
Yes, it could version